iFixit/react-commerce
# Troubleshooting: Use `id` values from server

This both ensures that jumplinks point at the right place and avoids the need to define a client-side function to generate ID values.

## QA Notes
Please confirm that jumplinks point at the right places on the page.
Here's an example page which links to another solution on the same page in the text of the first solution: https://react-commerce-git-troubleshooting-use-id-values-daf02c-ifixit.vercel.app/Vulcan/Dryer_Making_Loud_Noise

Closes https://github.com/iFixit/ifixit/issues/48525
